phpBB/styles/prosilver/template/ucp_pm_viewfolder.html

```diff
@@ -1,130 +1,73 @@
-				{L_NO_MESSAGES}
-			<!-- ENDIF -->
-		</strong></p>
-	<!-- ENDIF -->
-
-	<!-- IF FOLDER_CUR_MESSAGES neq 0 -->
-		<fieldset class="display-actions">
-			<div class="left-box"><label for="export_option">{L_EXPORT_FOLDER}{L_COLON} <select name="export_option" id="export_option"><option value="CSV">{L_EXPORT_AS_CSV}</option><option value="CSV_EXCEL">{L_EXPORT_AS_CSV_EXCEL}</option><option value="XML">{L_EXPORT_AS_XML}</option></select></label> <input class="button1 button button-form-bold" type="submit" name="submit_export" value="{L_GO}" /><br /></div>
-			<select name="mark_option">{S_MARK_OPTIONS}{S_MOVE_MARKED_OPTIONS}</select> <input class="button1 button button-form-bold" type="submit" name="submit_mark" value="{L_GO}" />
-			<div><a href="#" onclick="marklist('viewfolder', 'marked_msg', true); return false;">{L_MARK_ALL}</a> &bull; <a href="#" onclick="marklist('viewfolder', 'marked_msg', false); return false;">{L_UNMARK_ALL}</a></div>
-		</fieldset>
-
-		<hr />
-
-		<div class="action-bar bottom">
-			<!-- INCLUDE display_options.html -->
-			<input type="hidden" name="cur_folder_id" value="{CUR_FOLDER_ID}" />
-
-			<div class="pagination">
-				{TOTAL_MESSAGES}
-				<!-- IF .pagination -->
-					<!-- INCLUDE pagination.html -->
-				<!-- ELSE -->
-					 &bull; {PAGE_NUMBER}
-				<!-- ENDIF -->
+{% include 'ucp_header.html' %}
+
+{% if not PROMPT %}
+	{% include 'ucp_pm_message_header.html' %}
+{% endif %}
+
+{% if messagerow %}
+	<div class="pm">
+			<ul class="pm-list">
+				{% for message in messagerow %}
+					<li class="pm-item">
+						<span class="pm-primary">
+							{% if message.FOLDER_IMG_STYLE == "pm_unread" %}
+								<i class="fa fa-certificate fa-fw pm-unread" title="Unread message" aria-label="Unread message"></i>
+							{% endif %}
+
+							{% if message.PM_CLASS == "pm_marked_colour" %}
+								<i class="fa fa-star fa-fw pm-marked" title="You marked this message important!" aria-label="You marked this message important!"></i>
+							{% endif %}
+
+							<a href="{message.U_VIEW_PM}" title="{message.SUBJECT}" class="pm-title {{ message.FOLDER_IMG_STYLE == 'pm_unread' ? 'pm-unread' }}">{message.SUBJECT}</a>
+							<span class="pm-secondary">
+								<span class="pm-secondary-item pm-authour">
+									{% if message.PM_CLASS == "pm_foe_colour" %}
+										<i class="fa fa-ban fa-fw" title="{message.MESSAGE_AUTHOR} is your foe!" aria-label="{message.MESSAGE_AUTHOR} is your foe!"></i>
+									{% else %}
+										<i class="fa fa-user fa-fw" aria-hidden="true"></i>
+									{% endif %}
+
+									{% if S_SHOW_RECIPIENTS %}
+										{{ lang('MESSAGE_TO') }} {message.RECIPIENTS}
+									{% else %}
+										{message.MESSAGE_AUTHOR_FULL}
+									{% endif %}
+								</span>
+								<span class="pm-secondary-item pm-date">
+									<i class="fa fa-clock-o fa-fw" aria-hidden="true"></i>
+									{message.SENT_TIME}
+								</span>
+							</span>
+						</span>
+						<span class="pm-meta">
```

Comment:
I still believe this is undesired. Let's delete it and move checkboxes to the right side, making it possible to select specific messages, not only whole threads. Remove whole `.pm-meta` part.